modules/retail/pages/retail-uyuni-branchserver.adoc

```diff
@@ -22,21 +22,27 @@ Before you register {productname} branch servers and terminals to your {productn
 The products you need for this procedure are:
 
 [[opensuse-channels-cli]]
-[cols="1,1,1,1,1", options="header"]
+[cols="1,1,1,1,1,1,1,1", options="header"]
```

Comment:
Are you sure this cols count is correct? You should be able to add new rows without adjusting this? Might want to build and check.